### ISSUE TITLE
Production configuration recommendations

### ISSUE THREAD

**Issue by dangkaka** [cshum/imagor#347]
Hello,

Many thanks for building this service. This is what I have waited for a very long time.
I tried to use this lib to replace Thumbor, but I have difficulty finding the correct server and configuration setup. My setup always hangs after some time running.

Any recommendations on the production infrastructure and configuration?
1. Server size? Should we use a big server or multiple smaller servers?
2. Recommended configuration?

What I'm using:
1. Server: t3.large (2 CPUs, 8GB RAM), each server is running 3 containers.
2. Configuration:
```
S3_LOADER_BUCKET = "xxx"
S3_STORAGE_BUCKET = "xxx"
S3_RESULT_STORAGE_BUCKET = "xxx"
IMAGOR_SECRET = "xxx"
IMAGOR_REQUEST_TIMEOUT = "10s"
VIPS_CONCURRENCY = "4"
IMAGOR_PROCESS_CONCURRENCY = "4"
```

My problem: with this setup, after some time running, the server starts getting 408 timeout, some urls 408 all the time and it gets worse and worse, it seems the process was stuck at some points.

I appreciate any help. Thanks a lot.

**Comment by cshum** [comment]
Would you try: 

remove `VIPS_CONCURRENCY `

```
IMAGOR_PROCESS_CONCURRENCY=4
IMAGOR_PROCESS_QUEUE_SIZE=4
```

You may also change these numbers to see how result varies.

**Comment by cshum** [comment]
@dangkaka  would you test the docker build at master branch?
https://github.com/cshum/imagor/pkgs/container/imagor/89834697?tag=master

```
ghcr.io/cshum/imagor@sha256:6ebb0c613cc09ebebf525fea72bc20b8bff5b9ec8b326528db0b0adc46836807
```

**Comment by dangkaka** [comment]
yeah sure I will, thanks a lot @cshum

**Comment by dangkaka** [comment]
seem good so far @cshum . are we going to create release v1.4.4?

**Comment by cshum** [comment]
Cool lets do

**Comment by cshum** [comment]
Changes available in v1.4.4

**Comment by dangkaka** [comment]
many thanks @cshum
